bundles/com.zeligsoft.domain.ngc.ccm.descriptorgeneration/template/mainTransformAxcioma.ext

```diff
@@ -783,7 +783,37 @@ create PlanConnectionDescription createAsyncClientSideSendcConnection(InstanceDe
 		// AMI Fragment connections are never multiplex.
 		this.createEndPoint(false, false, "sendc_" + receptacle.name, compInstance)
 	};
-	
+
+/*
+* Depending on the given connectorType, return corresponding 'provides' port name
+*/
+cached String getConnectorProvidesPortName(String connectorType):

```

Comment:
I think it would be better this functions accepted the InterfacePort rather than forcing all callers to access the connectorType.name themselves:

```suggestion
cached String getConnectorUsesPortName(InterfacePort port):
    let connectorType = port.connectorType.name:
```